pkg/registry/apps/rest/storage_apps.go

```diff
@@ -63,9 +63,13 @@ func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorag
 		storage["deployments/scale"] = deploymentStorage.Scale
 	}
 	if apiResourceConfigSource.ResourceEnabled(version.WithResource("statefulsets")) {
-		statefulsetStorage, statefulsetStatusStorage := statefulsetstore.NewREST(restOptionsGetter)
-		storage["statefulsets"] = statefulsetStorage
-		storage["statefulsets/status"] = statefulsetStatusStorage
+		// enable scale subresource for v1beta1 for e2e testing
+		// we are not able to test v1beta2 as v1beta2 is disabled at master.go
+		// TODO (juntee): remove the subresource before 1.8 release
```

Comment:
> I plan to:
> (1) change e2e test's version from v1beta1 to v1beta2 BEFORE code freeze
> (2) replace e2e test with integration test sometime AFTER 1.8 release

SGTM. if the switch to integration test happens before code freeze, that works as well.